sdk/ai/azure-ai-projects/samples/agents/sample_agents_azure_ai_search.py

```diff
@@ -68,7 +58,7 @@
     message = project_client.agents.create_message(
         thread_id=thread.id,
         role="user",
-        content="Hello, send an email with the datetime and weather information in New York?",
+        content="Tell me about Bill and Mellie",
```

Comment:
For the sample, should we add instructions how to create index using Azure AI Search? Or add documentation for that.

Reply:
added link to file comment